recipe/build.sh

```diff
@@ -12,6 +12,6 @@ cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX \
       .
 
 ninja
-if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" ]]; then
+if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" || "${CROSSCOMPILING_EMULATOR}" != "" ]]; then
```

Comment:
```suggestion
if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" || "${CROSSCOMPILING_EMULATOR:-}" != "" ]]; then
```
Because of the `set -u` at the top of this file, this test fails on osx-arm64 without default expansion.

@h-vetinari, this may be worth integrating in the migrator in general?

Reply:
> this may be worth integrating in the migrator in general?

Sounds good! You'd have to patch the line [here](https://github.com/regro/cf-scripts/blob/74008603a69721abf41331cccbf30f2688ed1e5e/conda_forge_tick/migrators/cross_compile.py#L134). Can you send a PR?

Reply:
https://github.com/regro/cf-scripts/pull/5044 should take care of this for future migrations. We still need to make the change here as well (PR incoming).